homeassistant/components/rachio/switch.py

```diff
@@ -342,9 +337,12 @@ def schedule_is_enabled(self) -> bool:
     def turn_on(self, **kwargs) -> None:
         """Start this schedule."""
 
-        self._controller.rachio.schedulerule.start(self.schedule_id)
+        self._controller.rachio.schedulerule.start(self._schedule_id)
         _LOGGER.debug(
-            "Schedule %s started on %s", self.name, self._controller.name,
+            "Schedule %s started on %s for %s",
+            self.name,
+            self._controller.name,
+            self.duration,
```

Comment:
```suggestion
            self._duration,
```

Reply:
Just saw that, I'm removing that part from the log, its wasn't really necessary anyway.